src/Integrations/Goodbits/SettingsGoodbits.php

```diff
@@ -348,6 +361,9 @@ public function getSettingsGlobalData(): array
 			);
 		}
 
-		return $output;
+		return [
+			...$output,
+			...$this->settingsTroubleshooting->getOutputGlobalTroubleshooting(SettingsGoodbits::SETTINGS_TYPE_KEY),
```

Comment:
This is tested? Because array key unpacking is available since PHP 8.1

Reply:
yap it works for  Indexed arrays only at this point so no problem here